kiva/fonttools/font.py

```diff
@@ -98,21 +193,35 @@ class Font(object):
     def __init__(self, face_name="", size=12, family=SWISS,
                  weight=WEIGHT_NORMAL, style=NORMAL, underline=0,
                  encoding=DEFAULT):
-        if (not isinstance(face_name, str)
-                or not isinstance(size, int)
-                or not isinstance(family, int)
-                or not isinstance(weight, int)
-                or not isinstance(style, int)
-                or not isinstance(underline, int)
-                or not isinstance(encoding, int)):
+        if not isinstance(face_name, str):
+            raise RuntimeError(
+                f"Expected face name to be a str, got {face_name!r}")
+        if not isinstance(size, int):
+            raise RuntimeError(
+                f"Excected size to be an int, got {size!r}")
+        if not isinstance(family, int):
+            raise RuntimeError(
+                f"Excected family to be an int, got {family!r}")
+        if not isinstance(weight, int):
+            raise RuntimeError(
+                f"Excected weight to be an int, got {weight!r}")
+        if not isinstance(style, int):
+            raise RuntimeError(
+                f"Excected style to be an int, got {style!r}")
+        if not isinstance(underline, int):
+            raise RuntimeError(
+                f"Excected underline to be a int, got {underline!r}")
+        if not isinstance(encoding, int):
+            raise RuntimeError(
+                f"Excected encoding to be a bool, got {encoding!r}")
```

Comment:
```suggestion
                f"Expected encoding to be a bool, got {encoding!r}")
```

Reply:
The line `raise RuntimeError("Bad value in Font() constructor.")` below this one is unreachable.

Reply:
```suggestion
                f"Expected encoding to be n int, got {encoding!r}")
```